.github/workflows/ci.yml

```diff
@@ -123,14 +123,16 @@ jobs:
           fi
           changes_externals=${changes_externals//$'\n'/'\0'}
           changes_c_cpp=${changes_c_cpp//$'\n'/'\0'}
-          changes_md=${changes_md//$'\n'/'\0'}
-          echo "::set-output name=run-build::$build"
-          echo "::set-output name=run-check-licenses::$check_licenses"
-          echo "::set-output name=run-format-check-c-cpp::$format_check_c_cpp"
-          echo "::set-output name=run-format-check-md::$format_check_md"
-          echo "::set-output name=changes-externals::$changes_externals"
-          echo "::set-output name=changes-c-cpp::$changes_c_cpp"
-          echo "::set-output name=changes-md::$changes_md"
+          delimiter=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 16)
```

Comment:
Using a random delimiter due to this recommendation
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings